### PR TITLE
Helm refresh charts when repo has synced

### PR DESCRIPTION
**Pull request by nwmac**

### CODE REVIEW

File: custom-src/frontend/app/custom/helm/store/helm.effects.ts [hunk 1]
```diff
@@ -37,8 +38,39 @@ export class HelmEffects {
...
 
+  // Ensure that we refresh the charts when a repository finishes synchronizing
+  @Effect()
+  updateOnSyncFinished$ = this.actions$.pipe(
+    ofType<GetAllEndpointsSuccess>(GET_ENDPOINTS_SUCCESS),
+    flatMap(action => {
+      // Look to see if we have any endpoints that are sycnhronizing
+      let updated = false;
+      Object.values(action.payload.entities.stratosEndpoint).forEach(endpoint => {
+        if (endpoint.cnsi_type === 'helm' && endpoint.endpoint_metadata) {
```
Comment (richard-cox) [inline custom-src/frontend/app/custom/helm/store/helm.effects.ts]: This should probably be `HELM_ENDPOINT_TYPE`

File: custom-src/frontend/app/custom/helm/store/helm.effects.ts [hunk 2]
```diff
@@ -188,4 +220,42 @@ export class HelmEffects {
     };
   }
 
+  private checkSyncStatus() {
+    // Dispatch request
+    const url = `/pp/${this.proxyAPIVersion}/chartrepos/status`;
+    const requestArgs = {
+      headers: null,
+      params: null
```
Comment (richard-cox) [inline custom-src/frontend/app/custom/helm/store/helm.effects.ts:229]: Should this be `this.syncing`?

**Review by richard-cox — CHANGES_REQUESTED** [review]
Two minor comments, otherwise LGTM

File: custom-src/frontend/app/custom/helm/store/helm.effects.ts (hunk 2 above)
Comment (nwmac) [reply custom-src/frontend/app/custom/helm/store/helm.effects.ts:229]: Discussed with Richard and leave as is

**Pull request merged**